docs/en/success-guide/index.md

```diff
@@ -2,6 +2,10 @@
 layout: page
 title: Setup and Success Guide
 advertise: Setup and Success Guide
+lang: en
+nav_link: Setup Guide Contents
+nav_level: 1
+nav_order: 100
 ---
 
 Version 1.1, released 2017-03-06
```

Comment:
just remove this line, and put the version only in the en/index.md (About) file.